pkg/services/ngalert/provisioning/templates.go

```diff
@@ -5,8 +5,11 @@ import (
 	"errors"
 	"fmt"
 	"hash/fnv"
+	"sort"
 	"unsafe"
 
+	"golang.org/x/exp/maps"
```

Comment:
Can it be just `maps` since it's 1.23?

Reply:
I find maps.Keys in 1.23 a little bit awkward to use for maps. I will change, though.